.goreleaser.yaml

```diff
@@ -14,22 +14,22 @@
 
 project_name: getenvoy
 builds:
-- binary: getenvoy
-  ldflags: "-s -w -X github.com/tetratelabs/getenvoy/internal/version.Current={{.Version}}"
-  main: ./cmd/getenvoy/main.go
-  env:
-  - CGO_ENABLED=0
-  goos:
-    - linux
-    - darwin
-  goarch:
-    - amd64
+  - binary: getenvoy
+    ldflags: "-s -w -X github.com/tetratelabs/getenvoy/internal/version.Current={{.Version}}"
+    main: .
```

Comment:
cool thing is because we now re-use goreleaser in `make e2e`, we don't need to worry if this will break late during release.